config/initializers/oracle.rb

```diff
@@ -8,12 +8,12 @@
     Rails.configuration.active_record.schema_format = ActiveRecord::Base.schema_format = :ruby
 
     ActiveRecord::ConnectionAdapters::TableDefinition.prepend(Module.new do
-      def column(name, type, options = {})
+      def column(name, type, **options)
```

Comment:
Overridden method seems to be like this so I think we should also use the same
```suggestion
      def column(name, type, index: nil, **options)
```

Reply:
I think it doesn't matter in this case - we do not do anything with the `index` kwarg, and we pass all kwargs within `**options` (whether it contains `:index` or not). If the logic of our overridden method would somehow depend on the value of `index`, I'd agree. As it's not the case, I think it's just declaring an unused variable.